CHANGELOG.md

```diff
@@ -95,6 +95,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Added `OnStride` special Lua function for `AHumans` that is called whenever they stride (i.e. when their `StrideSound` is played). Like playing `StrideSound`, this does not happen when the AHuman is climbing.
 
+- Exposed broad range of sounds to Lua (R/W) through their relevant SoundContainers. For each class, these include:
+	- **Actor**: `BodyHitSound`, `PainSound`, `DeathSound`, `DeviceSwitchSound`, `AlarmSound`
+	
```

Comment:
Please clean up some of the odd formatting here
1. The empty line here
2. The tabs after the `:`s in some spots

Also, please get rid of the list markers (i.e. `-`s), it makes the text quite spread out instead of staying grouped (try viewing the file to see what I mean).

Reply:
Done, the list markers were just cos I didn't understand why it failed to include my newlines, and the spacing is caused by that whitespace incidentally, if the first elements in the list are separated it increases the spacing of the whole list apparently.